packages/server-dev/src/manager/processes/startWatchers.mjs

```diff
@@ -57,8 +55,10 @@ Watch <server-dir>/build/plugins/*
 - Restart server.
 */
 
-async function startWatchers(context) {
-  await Promise.all([configWatcher(context), envWatcher(context)]);
+function startWatchers(context) {
+  return async () => {
```

Comment:
If you are returning a method, maybe getStartWatchers is a better name? Not sure how it is used.